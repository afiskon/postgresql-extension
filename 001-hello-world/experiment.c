#include <postgres.h>
#include <port.h>
#include <catalog/pg_type.h>
#include <executor/spi.h>
#include <utils/builtins.h>
#include <utils/jsonb.h>
#include <sys/time.h>
#include <limits.h>
#include <string.h>

PG_MODULE_MAGIC;
PG_FUNCTION_INFO_V1(experiment_hello);

Datum
experiment_hello(PG_FUNCTION_ARGS)
{
	PG_RETURN_TEXT_P(cstring_to_text("hello"));
}
