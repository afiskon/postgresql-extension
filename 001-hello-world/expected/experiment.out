CREATE EXTENSION experiment;
SELECT experiment_hello();
 experiment_hello 
------------------
 hello
(1 row)

DROP EXTENSION experiment;
